src/Definition/Common.php

```diff
@@ -24,11 +32,17 @@ class Common
      * @param string $length
      * @param bool $nullable
      */
-    public function __construct($type, $length = null, $nullable = true)
+
+    public function __construct($type, $length = null, $nullable = true, $default = null, $format = null)
```

Comment:
I believe this also changes the ctor in the `Redshift` class, unfortunately a BC break. Might be worth refactoring to an array.